### PR TITLE
Add zh_Hant Localizations.

### PR DESCRIPTION
**Pull request by megabitsenmzq**
1. Added two "verbatim" I forgot to add in the previous PR.
2. zh_HK to zh_Hant.

### CODE REVIEW

**Comment by alienator88** [comment]
There's quite a few text strings that seem to have been removed, the ones marked with:
`"extractionState" : "manual"`

I had to add those in manually because they are not part of the Pearcleaner app directly, they are coming from my Swift package(AlinFoundation) and it couldn't extract all those automatically.

**Comment by megabitsenmzq** [comment]
I'll change this PR to include zh_Hant localizations later. Also, include your manual strings. 👍

**Comment by megabitsenmzq** [comment]
@alienator88 After some inspection, I found the current zh_HK translation seems doesn't include any words that are not used in Taiwan. (I'm not native, so it's just my perspective.) So, I think we can just use zh_HK as zh_Hant for now. I already changed my PR to do that.

zh_HK should fall back from zh_Hant automatically. Maybe we can let @changanmoon test this.

**Comment by changanmoon** [comment]
> @alienator88 After some inspection, I found the current zh_HK translation seems doesn't include any words that are not used in Taiwan. (I'm not native, so it's just my perspective.) So, I think we can just use zh_HK as zh_Hant for now. I already changed my PR to do that.
> 
> zh_HK should fall back from zh_Hant automatically. Maybe we can let @changanmoon test this.

@megabitsenmzq Thanks for your mention, but I have to disagree with your opinion.

Some points to clarify here:

i. My commits are not meant to replace your work. When it comes to localization, I mainly focus on `zh-Hans` and `zh-HK`, so I may not be fully familiar with `zh-Hant`.

To be honest, I initially decided to add the `zh-Hans` translation, but after seeing your comments in issue #83, it seems that you're ready to translate `zh-Hans` and `zh-Hant,` so I chose to translate `zh-HK,` and I don't want to conflict with your contributions.

ii. Contrary to what some people may think, while `zh-HK` is similar to `zh-Hant` in many cases, `zh-HK` still has some differences with `zh-Hant`. This is because Hong Kong and Taiwan don't share the same style and standard of traditional Chinese.

For spoken language, people in Taiwan speak Mandarin (國語), while in Hong Kong (and Macau), Cantonese (廣東話，粵語) is mainly used for communication.

However, although the written language is identical to the spoken language in Taiwan (so both are Mandarin Chinese), in Hong Kong, the written language (書面語) is not the same as Mandarin:

> The written language used in official and formal settings is a less standardized form of [Modern Standard Chinese](https://en.wikipedia.org/wiki/Modern_Standard_Chinese) that is read with Cantonese phonology. There is also a written language based on the vocabulary and grammar of spoken Cantonese known as [written Cantonese](https://en.wikipedia.org/wiki/Written_Cantonese). Although the "biliterate and trilingual" policy implies an absence of support for written Cantonese, it has gained popularity in news media where entertainment and local news are related. Written Cantonese is unintelligible to non-Cantonese speakers and is considered nonstandard by some educators despite its widespread usage in Hong Kong. Some have also credited written Cantonese for solving the challenges that standard written Chinese had faced in popular culture.
</br>(From Wikipedia: *[Languages of Hong Kong](https://en.wikipedia.org/wiki/Languages_of_Hong_Kong)*)

Thus, Apple has its practice of localizing `zh-HK`: using Hong Kong-style written language, the macOS interface is more native to Cantonese speakers. What I've done is try my own best to follow Apple's practice as much as possible, rather than making them available for Taiwanese Mandarin.

In conclusion, from my own perspective, I don't think it is appropriate to merge `zh-HK` translations into `zh-Hant`.

However, if you are unclear about what translation to choose for `zh-Hant`, I suggest that you check out the [Apple Localization Term Glossary](https://applelocalization.com) by Kishikawa Katsumi, which could be of great help to you.

**Comment by megabitsenmzq** [comment]
I mean, I'm not Taiwanese, even though I have some knowledge about their language habits. If somebody native could translate zh_Hant, then it's better than me.

I expected there should be some difference between your zh_HK translation and zh_TW. But after I checked your translation, I saw nothing that felt wrong. The zh_TW translation for this app should be pretty much the same. So, using your translation as zh_Hant should not be a big problem. 

However, it might be better to copy your translation to zh_Hant instead of using it directly. What do you think?

**Comment by changanmoon** [comment]
> I mean, I'm not Taiwanese, even though I have some knowledge about their language habits. If somebody native could translate zh_Hant, then it's better than me.
> 
> I expected there should be some difference between your zh_HK translation and zh_TW. But after I checked your translation, I saw nothing that felt wrong from my perspective. The zh_TW translation for this app should be pretty much the same. So, using your translation as zh_Hant should not be a big problem.
> 
> However, it might be better to copy your translation to zh_Hant instead of using it directly. I can do that too. What do you think? Even if it might not be 100% correct, it's better than nothing. @changanmoon

Don't get me wrong, I'm not saying you are Taiwanese. :)

There are some differences between `zh-HK` and `zh-Hant` indeed, so in this case, instead of just merging `zh-HK` into `zh-Hant` and discarding `zh-HK`, ***I prefer to keep both `zh-HK` and `zh-Hant`.***

However, for `zh-Hant`, I think you can modify some of my translations as you wish, because they may sound weird in Mandarin. (If I were you, I would reference the Apple Localization Term Glossary to make sure they follow Apple's style.)

---
p.s. For your information:

I've checked `zh-Hans` translation. [funsiyuan](https://github.com/funsiyuan) only translated `Pearcleaner/Resources/Localizable.xcstrings`.

There are still some strings need to be done.

螢幕截圖 2024-10-30 07 36 34

**Comment by alienator88** [comment]
I'm not super worried about the info plist translations. Mostly just the localizable catalog.

**Comment by megabitsenmzq** [comment]
I'll update my strings this week.
By the way, iOS/macOS now handles the space between Chinese characters and English words automatically, so we should not add space ourselves anymore.

**Comment by megabitsenmzq** [comment]
I changed the files to copy zh_HK to zh_Hant instead of using zh_HK as zh_Hant. I also added missing new strings to zh_Hans and zh_Hant. This should be ready for review now.

**Comment by alienator88** [comment]
Thank you @megabitsenmzq!
Would you be able to resolve the conflicts? I found a couple more new strings in my 3rd party package that only showed up with the Uninstall alert setting enabled.

**Comment by megabitsenmzq** [comment]
Done! I also translated the new strings.
